kythe/cxx/common/indexing/KytheGraphRecorder.h

```diff
@@ -45,7 +45,8 @@ enum class NodeKindID {
   kDoc,
   kTBuiltin,
   kMeta,
-  kDiagnostic
+  kDiagnostic,
+  kUsr
```

Comment:
minor: I think `kClangUsr` makes it clear that this is in a distinct namespace, particular given the potential confusion around the 'Usr' abbreviation.